python/doc/examples/meta_modeling/kriging_metamodel/plot_gpr_advanced.py

```diff
@@ -0,0 +1,332 @@
+"""
+Advanced Gaussian process regression
+====================================
+"""
+
+# %%
+#
+# In this example we will build a metamodel using Gaussian process regression of the :math:`x\sin(x)` function.
+#
+# We will choose the number of learning points, the basis and the covariance model.
```

Comment:
I think we agreed to directly move examples instead of adding new ones (ie the old ones should be removed)

Reply:
@jschueller It looks like we decided to duplicate all the scripts if I understand well the minutes of our last meeting (I double checked with @adutfoy )